tests/modules/datasets/test_share.py

```diff
@@ -1498,21 +1498,6 @@ def test_verify_items_share_request(db, client, user2, group2, share3_processed,
     assert status == ShareItemHealthStatus.PendingVerify.value
 
 
```

Comment:
Removed because this function does not exist anymore, but we will introduce more tests once the refactoring is done